tests/Microsoft.Bot.Builder.AI.LUIS.Tests/LuisV3OracleTests.cs

```diff
@@ -205,126 +134,129 @@ public async Task TraceActivity()
+                    Assert.NotNull(luisTraceInfo["luisResult"]);
+                    Assert.NotNull(luisTraceInfo["luisOptions"]);
+                    Assert.NotNull(luisTraceInfo["luisModel"]);
 
                     var recognizerResult = luisTraceInfo["recognizerResult"].ToObject<RecognizerResult>();
-                    Assert.AreEqual(recognizerResult.Text, utterance);
-                    Assert.IsNotNull(recognizerResult.Intents["SpecifyName"]);
-                    Assert.AreEqual(luisTraceInfo["luisResult"]["query"], utterance);
-                    Assert.AreEqual(luisTraceInfo["luisModel"]["ModelID"], AppId);
+                    Assert.Equal(utterance, recognizerResult.Text);
+                    Assert.NotNull(recognizerResult.Intents["SpecifyName"]);
+                    Assert.Equal(utterance, luisTraceInfo["luisResult"]["query"]);
+                    Assert.Equal(AppId, luisTraceInfo["luisModel"]["ModelID"]);
                 },
                 "luisTraceInfo")
                 .Send(utterance)
                 .AssertReply(botResponse, "passthrough")
                 .StartTestAsync();
         }
 
-        [TestMethod]
+        [Fact]
         public async Task Composite1() => await TestJson<RecognizerResult>("Composite1.json");
 
-        [TestMethod]
+        [Fact]
         public async Task Composite2() => await TestJson<RecognizerResult>("Composite2.json");
 
-        [TestMethod]
+        [Fact]
         public async Task Composite3() => await TestJson<RecognizerResult>("Composite3.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndBuiltin() => await TestJson<RecognizerResult>("ExternalEntitiesAndBuiltin.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndComposite() => await TestJson<RecognizerResult>("ExternalEntitiesAndComposite.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndList() => await TestJson<RecognizerResult>("ExternalEntitiesAndList.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndRegex() => await TestJson<RecognizerResult>("ExternalEntitiesAndRegex.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndSimple() => await TestJson<RecognizerResult>("ExternalEntitiesAndSimple.json");
 
-        [TestMethod]
+        [Fact]
         public async Task ExternalEntitiesAndSimpleOverride() => await TestJson<RecognizerResult>("ExternalEntitiesAndSimpleOverride.json");
 
-        [TestMethod]
+        [Fact]
         public async Task GeoPeopleOrdinal() => await TestJson<RecognizerResult>("GeoPeopleOrdinal.json");
 
-        [TestMethod]
+        [Fact]
         public async Task Minimal() => await TestJson<RecognizerResult>("Minimal.json");
 
         // TODO: This is disabled until the bug requiring instance data for geo is fixed.
-        // [TestMethod]
+        // [Fact]
```

Comment:
Maybe we can skip this test with the TODO as the reason.

Reply:
We did the same for this PR https://github.com/southworks/botbuilder-dotnet/pull/314/files#diff-ab9c3185422704bbf99a9cb1aff1a056R216

Reply:
Yeah, that sounded familiar to me. Ok, let's keep it like this to have consistency.